src/sagemaker/clarify.py

```diff
@@ -85,6 +98,22 @@
         SchemaOptional("excluded_columns"): [Or(int, str)],
         SchemaOptional("joinsource_name_or_index"): Or(str, int),
         SchemaOptional("group_variable"): Or(str, int),
+        SchemaOptional("time_series_data_config"): {
+            "target_time_series": Or(str, int),
+            "item_id": Or(str, int),
+            "timestamp": Or(str, int),
+            SchemaOptional("related_time_series"): Or([str], [int]),
+            SchemaOptional("static_covariates"): Or([str], [int]),
+            SchemaOptional("dataset_format"): And(
```

Comment:
Why is it optional?

Reply:
As discussed offline, we are leaving it defined in schema as optional for now.